app/models/move.rb

```diff
@@ -124,6 +124,12 @@ def person
     raise 'Attempt to Access to person!!!'
   end
 
+  def documents
+    # We need to make sure that we're returning Documents that are either for the current
+    # moves profile or the current move to support backwards compatibility.
+    Document.kept.where(move_id: id).or(Document.where(documentable: profile))
+  end
```

Comment:
Do we need the "backwards compatibility" once the document migration is complete?
if not we should add a reminder, like a TODO, right?

Reply:
I don't need the reminder as I have a ticket detailing that I'll do it later.